specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/CommonDefinitions.json

```diff
@@ -3586,6 +3590,105 @@
         }
       }
     },
+    "FunctionsDeployment": {
+      "description": "Configuration section for Azure Function app deployment.",
+      "type": "object",
+      "properties": {
+        "storage": {
+          "description": "Storage for deployed package used by the Azure Function app.",
+          "type": "object",
+          "properties": {
+            "type": {
+              "description": "Property to select Azure Storage Type. Available option: blobContainer.",
+              "type": "string"
+            },
+            "value": {
+              "description": "Property to set the URL for the selected Azure Storage Type. Example: For blobContainer, value will be like https://<storageAccountName>.blob.core.windows.net/<containerName>.",
+              "type": "string"
+            },
+            "authentication": {
+              "description": "Authentication method to access the storage for deployment.",
+              "type": "object",
+              "properties": {
+                "type": {
+                  "description": "Property to select Authentication Type to access the selected storage. Available options: SystemAssignedIdentity, UserAssignedIdentity, StorageAccountAccessKey.",
+                  "type": "string"
+                },
+                "value": {
+                  "description": "Property to set the value for the selected Authentication Type. Example: For SystemAssignedIdentity, do not set value. For UserAssignedIdentity, set resource ID of identity. For StorageAccountAccessKey, set name of the app setting that has the storage account access key.",
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "FunctionsAlwaysReadyConfig": {
+      "description": "Sets the number of 'Always Ready' instances for a specific Azure Function or a specific trigger type.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Either an Azure Functions trigger or an Azure Function name."
```

Comment:
>     "description": "Either an Azure Functions trigger or an Azure Function name." [](http://example.com/codeflow?start=5&length=82)

How will you know which it is supposed to be interpreted as?

Reply:
I have updated the description to specify the naming convention for triggers vs function names.